pkg/app/override.go

```diff
@@ -15,8 +15,63 @@
 
 package app
 
+import (
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+)
+
+const (
+	// overrideKind is the override resource type.
+	overrideKind = "ksonnet.io/app-override"
+	// overrideVersion is the version of the override resource.
+	overrideVersion = "0.1.0"
```

Comment:
Should we reuse `DefaultAPIVersion` from `schema.go`? Alternatively, should we be comparing to the loaded API version of the current `App` spec, rather than a separate hardcoded value?

Reply:
No, DefaultAPIVersion is the version for `app.yaml`. These files are of different types and can be versioned separately.